ui/component/socialShare/view.jsx

```diff
@@ -4,61 +4,119 @@ import React from 'react';
     const rewardsApproved = user && user.is_reward_approved;
     const OPEN_URL = 'https://open.lbry.com/';
     const lbryUrl = canonicalUrl ? canonicalUrl.split('lbry://')[1] : permanentUrl.split('lbry://')[1];
     const lbryWebUrl = lbryUrl.replace(/#/g, ':');
     const encodedLbryURL: string = `${OPEN_URL}${encodeURIComponent(lbryWebUrl)}`;
     const referralParam: string = referralCode && rewardsApproved ? `?r=${referralCode}` : '';
     const lbryURL: string = `${OPEN_URL}${lbryWebUrl}${referralParam}`;
-
-    const shareOnFb = __('Share on Facebook');
-    const shareOnTwitter = __('Share On Twitter');
+    const directUrl = generateDirectUrl(name, claimId);
 
     return (
       <React.Fragment>
-        <CopyableText label={__('LBRY Link')} copyable={lbryURL} noSnackbar />
-        <div className="">
-          <Button
-            icon={ICONS.FACEBOOK}
-            button="link"
-            description={shareOnFb}
-            href={`https://facebook.com/sharer/sharer.php?u=${encodedLbryURL}`}
-          />{' '}
+        <CopyableText label={__('LBRY Link')} copyable={lbryURL} />
+        <div className="section__actions">
           <Button
+            className="share"
+            iconSize={24}
             icon={ICONS.TWITTER}
-            button="link"
-            description={shareOnTwitter}
             href={`https://twitter.com/intent/tweet?text=${encodedLbryURL}`}
           />
+          <Button
+            className="share"
+            iconSize={24}
+            icon={ICONS.REDDIT}
+            title={__('Share on Facebook')}
+            href={`https://reddit.com/submit?url=${encodedLbryURL}`}
+          />
+          {IOS && (
+            // Only ios client supports share urls
+            <Button
+              className="share"
+              iconSize={24}
+              icon={ICONS.TELEGRAM}
+              title={__('Share on Telegram')}
+              href={`tg://msg_url?url=${encodedLbryURL}&amp;text=text`}
+            />
+          )}
+          <Button
+            className="share"
+            iconSize={24}
+            icon={ICONS.LINKEDIN}
+            title={__('Share on LinkedIn')}
+            href={`https://www.linkedin.com/sharing/share-offsite/?url=${encodedLbryURL}`}
+          />
+          <Button
+            className="share"
+            iconSize={24}
+            icon={ICONS.FACEBOOK}
+            title={__('Share on Facebook')}
+            href={`https://facebook.com/sharer/sharer.php?u=${encodedLbryURL}`}
+          />
+          {webShareable && !isChannel && (
+            <React.Fragment>
+              <Button
+                className="share"
+                iconSize={24}
+                icon={ICONS.EMBED}
+                title={__('Embed this content')}
+                onClick={() => this.setState({ showEmbed: !showEmbed })}
+              />
+              <Button
+                className="share"
+                iconSize={24}
+                icon={ICONS.MORE}
```

Comment:
The fact that the more icon only shows a direct link is a little weird. 

I'm not even sure we want to give that link, since I'm not sure we're promising embedding via it.

Reply:
I changed it to include LBRY URL and Download - what do you think?